main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// Message object
+type msg struct {
+	Message string
+}
+
+// / endpoint

```

Comment:
Read step 7 again. This should be `/hello` endpoint, not `/`. Also, add more details to your comment.